en/docs/references/use-a-connector-in-your-component.md

```diff
@@ -0,0 +1,59 @@
+# Use a Connector 
+
+A connector is a component you or another party have developed and exposed as an API in Choreo, which you published as a connector in the [Marketplace](../marketplace.md). A connector lets you share your APIs privately or publicly, promoting the reuse of APIs. Developers can use them when they design their components in the low-code view.
+
+## Use a connector in your component design
+When you design your components in the low-code view, you can search and add the connectors displayed in the Marketplace as follows:
+
+1. Open the component you are designing in the web editor and add a connector.
```

Comment:
"Web Editor" should be in title caps.

Reply:
should it? since its not a specific component, should we have it in caps?